packages/react/src/components/CTA/__stories__/CTA.stories.js

```diff
@@ -51,6 +52,7 @@ storiesOf('Components|CTA', module)
             heading: copy[1],
             cta: {
               href: urlBy[type],
+              src: ArrowRight20,
```

Comment:
This and others should technically be `cta: { icon: { src: ArrowRight20 } }`. This is to keep the data models consistent and allow for passing props down like `aria-label`.

Reply:
yep completely missed this - will fix